.github/workflows/quick-check.yml

```diff
@@ -1,14 +1,17 @@
-name: Pull request
+name: Quick check
 on:
   pull_request:
-    branches: ["**"]
+  push:
```

Comment:
I think this change is now causing the quick-check.yml workflow to execute twice for every PR, once for the push and one for the PR event. Example:

https://github.com/servo/servo/pull/29452

Reply:
Oh, yes indeed. Master branch should be excluded, as in mozjs repo.

Reply:
and all depedabot branches.